### ISSUE TITLE
Add virt-sparsify 

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/snc#68]
```
$ qemu-img info crc.qcow2 
image: crc.qcow2
file format: qcow2
virtual size: 31G (33285996544 bytes)
disk size: 9.0G
cluster_size: 65536
Format specific information:
    compat: 1.1
    lazy refcounts: true
    refcount bits: 16
    corrupt: false

$ virt-sparsify ./crc.qcow2 ./crc_sparse.qcow2 --tmp tmp/
[   0.1] Create overlay file in tmp/ to protect source disk
[   0.1] Examine source disk
 100% ⟦▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒⟧ --:--
[   5.9] Fill free space in /dev/sda2 with zero
[   6.7] Fill free space in /dev/sda3 with zero
 100% ⟦▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒⟧ 00:00
[  47.1] Copy to destination and make sparse
[  98.0] Sparsify operation completed with no errors.
virt-sparsify: Before deleting the old disk, carefully check that the 
target disk boots and works correctly.


$ qemu-img info crc_sparse.qcow2 
image: crc_sparse.qcow2
file format: qcow2
virtual size: 31G (33285996544 bytes)
disk size: 8.3G
cluster_size: 65536
Format specific information:
    compat: 1.1
    lazy refcounts: false
    refcount bits: 16
    corrupt: false

**Comment by cfergeau** [comment]
virt-sparsify was added in commit a76e8721362f929, and later removed in commit f0768092dc2f0ff

> This commit removes the call to virt-sparsify as virt-resize man page indicates it should take care of doing this.
I did not run extensive tests around this, so maybe it's not working as well as the man page says.

**Comment by praveenkumar** [comment]
>  it should take care of doing this.

Yes, as per man page => Virt-resize by default performs sparse copying.  This means that it does not copy blocks from the source disk which are all zeroes.  This improves speed and efficiency, but will produce incorrect results if the target disk image contains unzeroed data.

But somehow when I tried the sparse on final disk it does remove again some zeroed data. I did test on libvirt side the resulted disk image does worked as expected. I am not sure it that will be the case of other platform.
